example/lib/presentation/pages/calendar_event.dart

```diff
@@ -843,6 +849,21 @@ class _CalendarEventPageState extends State<CalendarEventPage> {
     }
   }
 
+  Text _availabilityToText(Availability availability) {
+    switch (availability) {
```

Comment:
I'd suggest creating a const map from Availability to string and then just doing a lookup instead of this switch

Reply:
Fixed